templates/shared/contextual_footers/_devices_newsletter_signup.html

```diff
@@ -10,7 +10,7 @@
         @media only screen and (min-width: 768px) { #mce-success-response { height: 90%; }}
         #mc_embed_signup input.mce_inline_error { border-color: red; }
       </style>
-      <form action="https://canonical.us3.list-manage.com/subscribe/post?u=56dac47c206ba0f58ec25f314&amp;id=97a67dea26" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="mc-embedded-subscribe-form validate" target="_blank" novalidate>
+      <form action="https://canonical.us3.list-manage.com/subscribe/post?u=56dac47c206ba0f58ec25f314&amp;id=97a67dea26" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="mc-embedded-subscribe-form validate" target="_blank" rel="noopener noreferrer" novalidate>
```

Comment:
This seems validate but very awkward. This should be replaced with a returning post or an background submit.